cli/scancommands.go

```diff
@@ -330,8 +331,16 @@ func AuditCmd(c *components.Context) error {
 		}
 	}
 	auditCmd.SetTechnologies(technologies)
+	threadsFlag, err := c.GetIntFlagValue(flags.Threads)
+	if err != nil {
+		return err
+	}
+	threads, err := commands.DetectNumOfThreads(threadsFlag)
```

Comment:
```suggestion
	threads, err := pluginsCommon.GetThreadsCount(c)
```
Use already existing func

Reply:
@attiasas The already existing func doesn't limit the number of threads. I'm using the function that was created for curation (DetectNumOfThreads) so maybethe limitation is important for curation?